version_import.py

```diff
@@ -0,0 +1,76 @@
+import bpy
+
+
+class Return:
```

Comment:
`ImportResult` might be a clearer name for this class, just because there could be other classes that we decide to call "result"